.ci/Jenkinsfile

```diff
@@ -312,6 +448,7 @@ def generateFunctionalTestStep(Map args = [:]){
   }
   tags += excludeNightlyTag
 
+  // TODO: Is this still relevant?
```

Comment:
I'd say so, as we have removed the `pullRequestFilter` attribute from the YAML file. What I'm not sure if we want to still keep that ability. In the past, we used it to separate Centos executions from Debian ones. Now that we run on VMs, maybe it's time to retire that ability